syntax = "proto3";

package rip_rpc_itf;
message Login_DoLogin_Prm {
	string user = 1;
	string pswd = 2;
	int64 code = 3;
}
message Login_DoLogin_Ret {
	bool p0 = 1;
	int64 p1 = 2;
}
message Login_Test1_Prm {
	repeated string a = 1;
	repeated int64 b = 2;
	repeated uint64 c = 3;
	repeated uint64 d = 4;
	repeated int32 e = 5;
}
message Login_Test1_Ret {
	repeated int64 p0 = 1;
}
message Login_Test2_Prm {
	string key = 1;
}
message Login_Test2_Ret {
	bool p0 = 1;
}
